spec/shopify_transporter/exporters/magento/product_exporter_spec.rb

```diff
@@ -32,14 +33,69 @@ module Magento
 
             expected_result = [
               {
-                product_id: 12345,
+                product_id: '12345',
+                type: 'configurable',
                 top_level_attribute: "an_attribute",
               },
             ]
 
-            exporter = described_class.new(store_id: 1, client: soap_client)
+            exporter = described_class.new(store_id: 1, soap_client: soap_client)
             expect(exporter.export).to eq(expected_result)
           end
+
+          it 'retrieves simple products from Magento using the SOAP API and injects parent_id' do
```

Comment:
We should add a test: `It does not inject parent_id` if parent_id does not exist.